cocos/2d/components/sprite.ts

```diff
@@ -180,7 +180,6 @@ export class Sprite extends UIRenderer {
      */
     @type(SpriteAtlas)
     @displayOrder(4)
-    @tooltip('i18n:sprite.atlas')
```

Comment:
Why remote it?

Reply:
Because it doesn't exist in i18n configuration.
See: https://github.com/cocos/cocos-engine/blob/8ae4dcf8a3f62c57310e0e25fa9a7b0717b942b5/editor/i18n/en/localization.js#L333

I added the tooltip in ui.js which is more suitable since it just matches the property and doesn't need to configure it in the source code (sprite.ts).